### DISCUSSION TITLE
Cannot post new issue for `brew` project.

### DISCUSSION THREAD

**Discussion by AmokHuginnsson** [Homebrew/discussions#2465] [Everyday usage] [answered]
I try to post new issue for `brew` project.
I have read CONTRIBUTING.md and I carefully followed all the steps and filled all template inputs and checked all the checkboxes.
When I try to post filled out form for a bug (I click "Submit new issue" button) a blank page appears and browser inspector shows "400 Bad Request".

I am at a loss what I could be doing wrong. I tried both Firefox and Google Chrome on Ubuntu 21.10.

Please advise.

Thank you.

**Comment by alebcay** [comment, marked as answer]
Can you try in an incognito/private window in either of the browsers you've used? Do you have any browser extensions installed in either of the two browsers? Do you have an ad blocker, system-wide proxy, or anonymization software in use?

In any case, this is an issue with the GitHub interface itself (Homebrew team does not administer any of the infrastructure that hosts the issue tracker). If looking into the above doesn't get you anywhere, I would advise you [open a support ticket with GitHub](https://support.github.com/contact) so that they can help troubleshoot the issue further.

**Reply by AmokHuginnsson** [reply]
Google Chrome is pristine on my system, no extensions and no configuration.
I do not have system-wide proxy nor anonymization software installed.
I will open a support ticket with GitHub.
Thank you for this advise.

**Comment by carlocab** [comment]
I'm seeing this issue too.

**Comment by AmokHuginnsson** [comment]
I have opened [a ticket](https://github.com/github/feedback/discussions/7788) at github/feedback.
I am waiting for an answer.

**Comment by abcbarryn** [comment]
I have the same issue. I have tried Google Chrome, Internet Explorer, Microsoft Edge, Mac Safari, and Firefox, all without success. :(

**Comment by luckman212** [comment]
What should we do here? I upvoted @AmokHuginnsson 's [ticket](https://github.com/github/feedback/discussions/7788) but after 11 days I see there's no response from GH. It's pretty bad that we can't post issues on the project.

**Reply by Bo98** [reply]
You're probably more likely to get a response via [GitHub Support's bug report form](https://support.github.com/contact/bug-report) rather than posting to GitHub/feedback which is mostly for feature suggestions.

**Reply by luckman212** [reply]
Thanks, done. GH ticket #1409253

**Comment by luckman212** [comment]
Apparently this is fixable by https://github.com/Homebrew/discussions/discussions/2526#discussioncomment-1731168
